### PR TITLE
Required JS assignments

### PR DESCRIPTION
**Pull request by amycham**
calculator and countdown

### CODE REVIEW

File: Lesson03/exercise-one.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//create array min 5 strings
+let myArray = ['This', 'is', 'a', 'practice', 'array'];
+console.log(myArray);
+
+//add element end of array
+myArray.push('script');
+console.log(myArray);
+
+//remove third element
+myArray.splice(2,1);
```
Comment (jimmynono) [inline Lesson03/exercise-one.js]: Yes! Perfect. A space after the comma would be nice, but great job at using zero-indexing.

Comment (amycham) [reply Lesson03/exercise-one.js]: Haha! Would you believe I had one in there and took it out to more exactly match the spec? :-D

File: Lesson03/exercise-three.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+//create array w/at least 5 strings
+let myArray = ['hot dogs', 'french fries', 'goldfish', 'strawberries', 'blueberries', 'broccoli'];
+
+//get array length
+let arrLength = myArray.length;
+
+//iterate with for loop
+for(let i=0; i<arrLength; i++){
+   //console log each element plus index, sep with comma
+  console.log(`${i},${myArray[i]}`);
```
Comment (jimmynono) [inline Lesson03/exercise-three.js]: string concatenation! LOVE IT

File: Lesson03/exercise-three.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+//create array w/at least 5 strings
+let myArray = ['hot dogs', 'french fries', 'goldfish', 'strawberries', 'blueberries', 'broccoli'];
+
+//get array length
+let arrLength = myArray.length;
+
+//iterate with for loop
+for(let i=0; i<arrLength; i++){
```
Comment (jimmynono) [inline Lesson03/exercise-three.js]: Missing spaces around the = sign

File: Lesson03/exercise-one.js (hunk 1 above)
Comment (jimmynono) [reply Lesson03/exercise-one.js]: Ha!

File: Lesson02/cardDeck.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+let suits = ["hearts", "diamonds", "clubs", "spades"];
+let faces = ["Ace", "2", "3", "4", "5", "6", "7", "8", "9", "10", "Jack", "Queen", "King"];
+
+let lenS = suits.length;
```
Comment (jimmynono) [inline Lesson02/cardDeck.js]: You don't need these variables, you could simply use the length check in the for loop.
```
for (let i = 0; i < suits.length; i++) {
```

**Pull request closed without merging**